src/VariableSizeList.js

```diff
@@ -147,6 +147,7 @@ const getEstimatedTotalSize = (
   let totalSizeOfMeasuredItems = 0;
 
   if (lastMeasuredIndex >= 0) {
+    if (lastMeasuredIndex >= itemMetadataMap.length) lastMeasuredIndex = itemMetadataMap.length - 1;
```

Comment:
[`itemMetadataMap` is an `Object`](https://github.com/bvaughn/react-window/blob/a22f3e983dd3d15da1e51230189c928ad93f9bbb/src/VariableSizeList.js#L252) so it doesn't have a `length` property.

Reply:
I'll just make this adjustment myself and also add the fix to `VariableSizeGrid`. Thanks for pointing this out.